src/sql/workbench/services/notebook/common/localContentManager.ts

```diff
@@ -272,6 +272,7 @@ namespace v3 {
 					name: name,
 					text: v4.demultiline(output.text)
 				};
+			case 'error':
```

Comment:
An error output can have either an 'error' or 'pyerr' output type, but we missed 'error' here. Without this line there's a build failure from trying to convert an Error output to nb.ICellOutput in the default return statement below.